src/index.js

```diff
@@ -21,7 +21,7 @@ const matches = (key, importee) => {
   return importeeStartsWithKey && importeeHasSlashAfterKey;
 };
 const endsWith = (needle, haystack) => haystack.slice(-needle.length) === needle;
-const isFilePath = id => /^\.?\//.test(id);
+const isFilePath = id => /^\.{0,2}\//.test(id);
```

Comment:
but why?

Reply:
Currently `{find:'something', replacement: '../../../something'}` is not working as expected because `isFilePath` does not think `'../../../something'` is a file path. Thus the plugin does not resolve the path, which causes problems.